libcontainer/standard_init_linux.go

```diff
@@ -79,6 +79,14 @@ func (l *linuxStandardInit) Init() error {
 	}
 
 	label.Init()
+
+	// Use Unshare() to enter a new cgroup_namespace before InitializeMountNamespace()
+	if l.config.Config.Namespaces.Contains(configs.NEWCGROUP) && l.config.Config.Namespaces.PathOf(configs.NEWCGROUP) == "" {
+		if err := syscall.Unshare(configs.Syscall_CLONE_NEWCGROUP); err != nil {
```

Comment:
Haha, yeah, I know. I'm going to play around with it myself first. I would turn to you for help without hesitation if I have any questions. Thanks in advance :stuck_out_tongue_winking_eye: